packages/babel-plugin-minify-simplify/__tests__/fixtures/bail-on-different-member-exprs/actual.js

```diff
@@ -0,0 +1,5 @@
+if (a) {
+  this.a = 1;
+} else {
+  this.b = 2;
+}
```

Comment:
the filename looks wrong. It doesn't bail here. `if-else-member-to-conditional` ?